.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npx lint-staged
+# npx lint-staged
```

Comment:
Why is this change here?

Reply:
will re-surrect, my cygwin does not have a npx installed